pkg/console/operator/status.go

```diff
@@ -61,6 +61,50 @@ const (
 	reasonCustomLogoInvalid   = "CustomLogoInvalid"
 )
 
+// handleDegraded can be used to set a number of Degraded statuses
+// example:
+//   co.handleDegraded(operatorConfig, "RouteStatus", err)
+//   co.handleDegraded(operatorConfig, "CustomLogoInvalid", err)
+// creates condition:
+//   RouteStatusDegraded
+//   CustomLogoInvalidDegraded
+// and uses the error as its message
+func (c *consoleOperator) HandleDegraded(operatorConfig *operatorsv1.Console, prefix string, err error) {
```

Comment:
Not really sure if it wouldn't be better to have enum instead of plane prefix., so we have all the various conditions under control. 
Here is an [example](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/3697805493449a5affd78610b20ccf55cdb82eed/pkg/operator/workloadcontroller/workload_controller.go#L35) even though they have only one...

Reply:
I'm following along the lines of this:
https://github.com/openshift/cluster-authentication-operator/pull/142/files

Reply:
I don't believe any of the strings should be reused. Each one should be specific enough for a single use in one place.

Reply:
> I don't believe any of the strings should be reused. Each one should be specific enough for a single use in one place.

sounds legit 👍

Reply:
That way we can `grep` to find the point of failure & know for certain what is going wrong.  In theory anyway...